Abstract/portable-formula.rb

```diff
@@ -49,6 +58,21 @@ def archs
       return [Hardware::CPU.arch_32_bit]
     end
 
+    # On Tiger and Leopard, override the default behaviour.
+    # Normally we don't build 64-bit there, because the linker is
+    # temperamental, and so "universal" builds don't usually happen.
+    # However, for the purposes of distributing portable packages,
+    # it's very useful to be able to build i386/ppc binaries for use
+    # on both Intel and PowerPC Macs. The Apple-provided compilers
+    # are capable of this on both Intel and Mac hosts.
+    if MacOS.version < :snow_leopard
+      if build.with? "universal"
+        return [:i386, :ppc]
+      else
+        return [Hardware::CPU.arch_32_bit]
```

Comment:
It seems that two different styles are used here:
* `Hardware::CPU.universal_archs/Hardware::CPU.arch_64_bit/Hardware::CPU.arch_32_bit` 
* `:x86_64/:i386/:ppc`

I wonder would it better to enforce only one style here. At least, this will make it easy to read.